crates/typst-library/src/foundations/array.rs

```diff
@@ -154,55 +151,53 @@ impl Array {
 
 #[scope]
 impl Array {
-    /// Converts a value to an array.
+    /// 値を配列に変換します。
     ///
-    /// Note that this function is only intended for conversion of a collection-like
-    /// value to an array, not for creation of an array from individual items. Use
-    /// the array syntax `(1, 2, 3)` (or `(1,)` for a single-element array) instead.
+    /// この関数は、個々の要素から配列を作成するためのものではなく、コレクションのような値を配列に変換する
+    /// ことのみを目的としています。個々の要素から配列を作成する場合は、配列の構文である (1, 2, 3)
+    ///  (または、単一要素の配列の場合は (1,)) を使用してください。
```

Comment:
`または〜`を囲う括弧は全角のほうがよいかもしれません。
```suggestion
    ///  （または、単一要素の配列の場合は (1,)）を使用してください。
```